src/stylesheets/elements/_links.scss

```diff
@@ -22,7 +22,24 @@ a {
 /*--- Figure ---*/
 
 figure a {
```

Comment:
Doesn't look like this any of the focus and hover is actually making a difference. Poential to remove all of this but do test

Reply:
This also applies to links within figcaption. Not sure if we should:
- move it to figcaption
- remove it altogether

Reply:
The media figcaption looks the same as our traditional links. Can you find all instances of figure and figcaption within the apps. If it is just used in the media, list, and hybrid bills views then it would fine to delete.